### ISSUE TITLE
[NV-3048] Update invite members icon when a user click on the avatar.

### ISSUE THREAD

**Issue by maverox** [novuhq/novu#4601]
📜 Description
We can have a different icon for the invite members button

👟 Reproduction steps
Click on the profile icon at the top right
Small Icon with >invite Members button
📸ScreenShot
![Screenshot 2023-10-20 194629](https://github.com/novuhq/novu/assets/29498864/837b87cc-5a49-4e99-96e3-302789e45439)

👍 Expected behavior
Please implement the changes according to [this Figma designs](https://www.figma.com/file/mxViLXxtC0PnB0ehXDcafw/Crafting-updates?type=design&node-id=742-1491&mode=design&t=zxBADQISFjfH2Yvb-0)

👎 Actual Behavior with Screenshots
Novu version
0.18.0

npm version
No response

node version
No response

📃 Provide any additional context for the Bug.
👀 Have you spent some time to check if this bug has been raised before?
 I checked and didn't find a similar issue
🏢 Have you read the Contributing Guidelines?
 I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)
Are you willing to submit PR?
Yes I am willing to submit a PR!

<sub>[NV-3048](https://linear.app/novu/issue/NV-3048/update-invite-members-icon-when-a-user-click-on-the-avatar)</sub>

**Comment by Priyanshuthapliyal2005** [comment]
can i work on this issue

**Comment by maverox** [comment]
> can i work on this issue

hey, I am also waiting to be assigned on this, that's why i raised this issue
the thing is we have to wait for the maintainers to check on this issue and then assign it.

**Comment by p-fernandez** [comment]
@maverox closing this as it is a duplicate of this one: https://github.com/novuhq/novu/issues/4111

**Comment by p-fernandez** [comment]
Reopening again as I thought it was tackled in the other PR. Assigning to you @maverox

**Comment by Sukomal07** [comment]
@p-fernandez please review my PR

**Comment by maverox** [comment]
@p-fernandez  can you please review my PR #4622

**Comment by p-fernandez** [comment]
@Sukomal07 thank you for the effort because you did a good PR. Unfortunately the issue was assigned to @maverox so I had to close your PR in favour of his.
Please next time, before starting to work in any issue ask to get is assigned to avoid working in things other contributor is working. We have multiple issues and opportunities to collaborate with so take a look on any that is unassigned that might catch your interest. Thank you.

**Comment by Sukomal07** [comment]
@p-fernandez Thank you, when i start work on this issue noone is assigned. Bad luck for me 😄

**Comment by maverox** [comment]
@p-fernandez  can you please review PR #4644

**Comment by maverox** [comment]
@p-fernandez  please review pr #4693
